aospy/test/test_calc_basic.py

```diff
@@ -143,5 +146,55 @@ def setUp(self):
             'dtype_out_vert': 'vert_int'
         }
 
+class TestCalcAttrs(unittest.TestCase):
+    def test_units_description_attrs(self):
```

Comment:
Building off of @spencerahill's comment regarding using `pytest`, this is a classic use case for [`pytest.mark.parametrize`](https://docs.pytest.org/en/latest/parametrize.html), so let's make sure to use that too.  See here for an example use in our code-base: https://github.com/spencerahill/aospy/blob/2ef0b9a90a3752db229d51fb531a5387801dcd41/aospy/test/test_automate.py#L144-L150

The benefit of using `pytest.mark.parametrize` is that it splits up this big group of tests (which if one fails the entire test function fails, and the remaining cases don't get run) into individual tests, each of which gets run separately.  This gives us a more complete picture for debugging if something goes wrong.

Reply:
that's cool. I'll use that.